com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/persistence/dto/OperationDto.java

```diff
@@ -81,7 +81,7 @@ private OperationDto(String processId, String processType, Date startedAt, Date
 
     @Override
     public String getPrimaryKey() {
-        return processId;
+        return getProcessId();
```

Comment:
The two changes are for different reasons.
This was thanks to a sonar issue, as it stated that this method and `getProcessId` did the same thing. So I thought that the "primary key" should get that "primary" property from the concrete object (as the `getPrimaryKey` method is from the interface)
The change in DeployedComponents was because there was no need to use a getter on a field from the same object as long as the getter doesn't do any extra processing.